client/elements/sc-site-layout.js

```diff
@@ -147,25 +147,16 @@ export class SCSiteLayout extends LitLocalized(LitElement) {
       this.actions.changeDisplaySuttaInfoState(false);
     });
 
-    let scrollDistance = 0;
+    // handles scroll-based universal toolbar animation on home page
```

Comment:
As far as I can tell, this listener is only meant to handle the toolbar animation on the homepage. 

I added a comment to make that explicit and an early return if not on the home page. Please let me know if that is correct.

Reply:
Works great, this is something I've always wanted to improve, thanks :pray:

Reply:
:+1: